Alignment/OfflineValidation/plugins/OverlapValidation.cc

```diff
@@ -383,7 +383,7 @@ void OverlapValidation::analyzeTrajectory(const Trajectory& trajectory,
     ++overlapCounts_[1];
     if ((layer != -1) && (acceptLayer[subDet])) {
       for (vector<TrajectoryMeasurement>::const_iterator itmCompare = itm - 1;
-           itmCompare >= measurements.begin() && itmCompare > itm - 4;
+           itmCompare >= measurements.begin() && itmCompare - (itm - 4) > 0;
```

Comment:
wouldn't 
```C++
itmCompare > (itm - 4);
```

work as well and be slightly more readable?

Reply:
for my education how do you compile with this flag?

Reply:
just use CMSSW_12_5_LTO_X IBs

Reply:
how about
```
(itmCompare >= measurements.begin()) && ((itmCompare > (itm - 4)) > 0);
```

Reply:
This also works :)